lib/xcpretty/parser.rb

```diff
@@ -193,7 +193,7 @@ module Warnings
 
       # @regex Captured groups
       # $1 = whole warning
-      GENERIC_WARNING_MATCHER = /^warning:\s(.*)$/
```

Comment:
Unfortunately we can't do this anymore, as it's a significant performance regression.
Is there a sample warning you could paste that doesn't conform to this one?